packages/esm-outpatient-app/src/queue-patient-linelists/queue-linelist-base-table.component.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  DataTable,
+  DataTableHeader,
+  OverflowMenu,
+  OverflowMenuItem,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableToolbar,
+  TableToolbarContent,
+  TableToolbarSearch,
+  Tile,
+  Button,
+  Tag,
+} from 'carbon-components-react';
+import { navigate, showModal, ExtensionSlot, formatDatetime, usePagination } from '@openmrs/esm-framework';
+import { getOriginFromPathName } from '../active-visits/active-visits-table.resource';
+import styles from './queue-linelist-base-table.scss';
+import OverflowMenuVertical16 from '@carbon/icons-react/es/overflow-menu--vertical/16';
+import { Filter16 } from '@carbon/icons-react';
+
+const currentPathName: string = window.location.pathname;
+const fromPage: string = getOriginFromPathName(currentPathName);
+
+type FilterProps = {
+  rowIds: Array<string>;
+  headers: Array<DataTableHeader>;
+  cellsById: any;
+  inputValue: string;
+  getCellId: (row, key) => string;
+};
+
+interface QueuePatientTableProps {
+  title: string;
+  patientData: Array<any>;
+  headers: Array<any>;
+  rows: any;
+  serviceType: string;
+}
+
+function ActionsMenu({ patientUuid }: { patientUuid: string }) {
+  const { t } = useTranslation();
+
+  const launchEndVisitModal = useCallback(() => {
+    const dispose = showModal('end-visit-dialog', {
+      closeModal: () => dispose(),
+      patientUuid,
+    });
+  }, [patientUuid]);
+
+  return (
+    <OverflowMenu light selectorPrimaryFocus={'#editPatientDetails'} size="sm" flipped>
+      <OverflowMenuItem className={styles.menuItem} id="#outreachFrom" itemText={t('outreachFrom', 'Outreach From')}>
+        {t('outreachFrom', 'Outreach From')}
+      </OverflowMenuItem>
+      <OverflowMenuItem className={styles.menuItem} id="#locatorFrom" itemText={t('locatorFrom', 'locatorFrom')}>
+        {t('locatorFrom', 'Locator From')}
+      </OverflowMenuItem>
+      <OverflowMenuItem
+        className={styles.menuItem}
```

Comment:
I think for now we can remove the outreach form and locator  form from the overflow menu for scheduled appointment list.